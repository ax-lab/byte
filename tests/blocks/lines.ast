#(
	Simple line parsing
	===================
)

line 1
line 2
line 3

line 4

# Comments are ignored
line 5
