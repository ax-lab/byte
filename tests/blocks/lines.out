Line(
	line
	1
)
Line (
	line
	2
)
Line (
	line
	3
)
Line (
	line
	4
)
Line (
	line
	5
)
